Sprint-2/debug/1.js

```diff
@@ -1,8 +1,19 @@
 // Predict and explain first...
+// Prediction : it will return "the sum of 10 and 32 is null". Because the function returns something but it's null.
```

Comment:
`undefined` is not the same as `null` in JS. So which value is returned from the original function?